service-front/module/Application/config/module.config.php

```diff
@@ -47,6 +47,26 @@
                             ],
                         ],
                     ],
+                    'health_check' => [
+                        'type' => Literal::class,
+                        'options' => [
+                            'route' => '/health-check',
+                            'defaults' => [
+                                'controller' => Controller\IndexController::class,
+                                'action' => 'healthCheck',
+                            ],
+                        ],
+                    ],
+                    'health_check_service' => [
+                        'type' => Literal::class,
+                        'options' => [
+                            'route' => '/health-check-service',
```

Comment:
To match [the standards](https://docs.opg.service.justice.gov.uk/documentation/adrs/adr-007.html#context):
```suggestion
                            'route' => '/health-check/service',
```

Reply:
This still needs to be updated